boost-gradle/src/main/groovy/io/openliberty/boost/gradle/utils/GradleProjectUtil.groovy

```diff
@@ -37,4 +40,33 @@ public class GradleProjectUtil {
 
         return version        
     }
+    
+    public static List<String> getSpringFrameworkDependencies(Project project) {
+
+        List<String> springBootDependencies = new ArrayList<String>()
+
+        project.configurations.compile.resolvedConfiguration.resolvedArtifacts.collect { it.moduleVersion.id }.each { ModuleVersionIdentifier id ->
+        
+            if ( id.group.equals("org.springframework") ) {
+                
```

Comment:
This isn't a "startsWith"  of "org.springframework"  ???

Reply:
I know we addressed this in gitter, but just for consistency here, the spring artifacts we are interested in have "org.springframework" as the full groupID. This is acting as a filter here.